packages/ember-cli-fastboot/addon/services/fastboot.js

```diff
@@ -69,6 +69,13 @@ const FastBootService = Service.extend({
   headers: deprecatingAlias('request.headers', { id: 'fastboot.headers-to-request', until: '0.9.9' }),
   isFastBoot: typeof FastBoot !== 'undefined',
 
+  isFastboot: computed(function() {
```

Comment:
Or should we warn here instead and return `this.isFastBoot`?

Reply:
I think an assertion is fine, though I'd prefer if we didn't make this a computed (it will only assert the first time it is accessed I think).

We should consider migrating to native classes here, and then using a native getter directly. Unfortunately, we'll have to do this conditionally based on Ember version. We need to drop Ember < 3.12 support, would you mind making an issue for discussing that?

Reply:
Created https://github.com/ember-fastboot/ember-cli-fastboot/issues/815 for this. Should I just leave the PR until using a native getter is possible or?